### PR TITLE
efibootmgr: make modifying of boot records optional, disabled by default.

### PR DESCRIPTION
**Pull request by CMB**
Also changed the name of the config file, since the syntax
is much different.

See issue #5165 for the discussion.

### CODE REVIEW

File: srcpkgs/efibootmgr/INSTALL.msg [hunk 1]
```diff
@@ -1,11 +1,12 @@
-From now on, efibootmgr automatically generates efi boot entries to
+efibootmgr can automatically generate EFI boot entries to boot
 directly boot the kernel after every kernel update or installation.
```
Comment (aeosynth) [inline srcpkgs/efibootmgr/INSTALL.msg]: > to boot directly boot

the added 'boot' is unnecessary

File: srcpkgs/efibootmgr/template [hunk 2]
```diff
@@ -23,7 +23,7 @@ do_install() {
 post_install() {
 	vlicense COPYING
 	vmkdir etc/
-	touch ${DESTDIR}/etc/efibootmgr-options.conf
+	vinstall "${FILESDIR}/efibootmgr-kernel-hook.conf" 644 etc/
```
Comment (Vaelatern) [inline srcpkgs/efibootmgr/template]: Please use `vconf`

File: srcpkgs/efibootmgr/files/efibootmgr-kernel-hook.conf [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# Options for the kernel hook script installed by the efibootmgr package.
+modify_efi_entries=0
```
Comment (Vaelatern) [inline srcpkgs/efibootmgr/files/efibootmgr-kernel-hook.conf]: Please change these options to be all-caps.

**Comment by CMB** [comment]
Fixed to use vconf.
Options in the config file are all-caps.
Also added braces around variable names in a couple places.
Typo in the install message is fixed.

Thanks for the reviews.

**Comment by Vaelatern** [comment]
@Duncaen Should the conf file be moved? To `/etc/default/` seems reasonable, but I am not sure.
the problem is once this gets merged, moving a file like this will be more problematic.

**Pull request merged**